pkgs/modules/packagers/python/default.nix

```diff
@@ -0,0 +1,72 @@
+{ pkgs, lib, config, ... }:
+with pkgs.lib;
+let
+  cfg = config.packagers.python;
+  pythonVersion = config.interpreters.python.version;
+  python = pkgs.python-versions.${pythonVersion}.python;
+  pypkgs = pkgs.python-versions.${pythonVersion}.pythonPackages;
+  pylibs-dir = ".pythonlibs";
+  pip = import ./pip.nix (pkgs // {
+    inherit python pypkgs;
+  });
+  pythonUtils = import ../../../python-utils {
+    inherit pkgs python pypkgs;
+  };
+  pythonWrapper = pythonUtils.pythonWrapper;
+  pip-wrapper = pythonWrapper { bin = "${pip.pip}/bin/pip"; name = "pip"; };
+  poetry = pkgs.callPackage (../../../poetry/poetry-py + "${pythonVersion}.nix") {
+    inherit python;
+    inherit pypkgs;
+  };
+  poetry-config = pkgs.writeTextFile {
+    name = "poetry-config";
+    text = ''
+  '';
+    destination = "/config.toml";
+  };
+  poetry-wrapper = pythonWrapper { bin = "${poetry}/bin/poetry"; name = "poetry"; };
+  versionSupported = pythonVersion != "3.8";
+in
+{
+  options = {
+    packagers.python = {
+      enable = mkModuleEnableOption {
+        name = "Python packager";
+        description = "Python packaging with Poetry or Pip";
+      };
+    };
+  };
+
+  config = mkIf (cfg.enable && versionSupported) {
```

Comment:
Here we forcibly disable packager for Python 3.8 because pip doesn't build for Python 3.8 in unstable and I don't bother fixing it.

Reply:
But the rest of the python bundle still can work for 3.8!

Reply:
3.8 is officially EOL 2024-10, so only 6 more months until more of the ecosystem stops maintaining backwards compatibility